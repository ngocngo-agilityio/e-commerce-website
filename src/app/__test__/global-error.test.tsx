import { render } from '@testing-library/react';

// Components
import GlobalError from '../global-error';

// Mocks
import { MOCK_ERROR_MESSAGES } from '@mocks';

describe('HomeLoading', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <GlobalError error={new Error(MOCK_ERROR_MESSAGES)} reset={jest.fn()} />,
    );

    expect(container).toMatchSnapshot();
  });
});
