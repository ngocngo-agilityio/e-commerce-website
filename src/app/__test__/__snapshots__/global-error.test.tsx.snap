// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLoading should match snapshot 1`] = `
<div>
  <html>
    <body>
      <div
        class="chakra-stack css-1ts0oy9"
      >
        <h2
          class="chakra-heading css-0"
        >
          Something went wrong!
        </h2>
        <p
          class="chakra-text css-tokvmb"
        >
          Request failed
        </p>
        <button
          class="chakra-button css-4xx2wk"
          type="button"
        >
          Try again
        </button>
      </div>
    </body>
  </html>
</div>
`;
